src/modules/fancyzones/lib/FancyZonesData.h

```diff
@@ -0,0 +1,137 @@
+#pragma once
+
+#include "JsonHelpers.h"
+
+#include <common/settings_helpers.h>
+#include <common/json.h>
+#include <mutex>
+
+#include <string>
+#include <unordered_map>
+#include <optional>
+#include <vector>
+#include <winnt.h>
+
+namespace FancyZonesDataTypes
+{
+    struct ZoneSetData;
+    struct DeviceInfoData;
+    struct CustomZoneSetData;
+    struct AppZoneHistoryData;
+}
+
+#if defined(UNIT_TESTS)
+namespace FancyZonesUnitTests
+{
+    class FancyZonesDataUnitTests;
+    class FancyZonesIFancyZonesCallbackUnitTests;
+    class ZoneSetCalculateZonesUnitTests;
+    class ZoneWindowUnitTests;
+}
+#endif
+namespace FancyZonesData
```

Comment:
Do we need this namespace?
It's used mainly for `FancyZonesData::FancyZonesDataInstance()`, I don't see a significant value in using it.
Also, `FancyZonesDataTypes` is a separate namespace, and seems inconsistent to have two different namespaces for two things that are strictly related.
Keep `FancyZonesDataTypes` and remove `FancyZonesData`

Reply:
Done